packages/ui/src/utils/HelperUtils.ts

```diff
@@ -15,3 +16,43 @@ export function getHostName(url: string) {
 
   return newUrl.hostname
 }
+
+export function generateAvatarColors(address: string): string {
```

Comment:
Do we need to specify return type of `: string` here? As it will be inherited by typescript, in general I tend to let ts do all the heavy lifting and only specify types when those are inherited as `any` only.